rerun_py/rerun_sdk/rerun/datatypes/aabb2d_ext.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+if TYPE_CHECKING:
+    from . import Vec2DLike
+
+
+class AABB2DExt:
```

Comment:
why is an extension for this needed, codegen should do the right thing here?

Reply:
looks like on the _component_ you wanted to force the parameters to be named, but here you forgot?

Reply:
I have no idea. This is Python, and I am just copy-pasting from other files.

Who is responsible for converting Vec2DLike into a numpy array?